src/main/java/baseball/utils/Parser.java

```diff
@@ -0,0 +1,17 @@
+package baseball.utils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class Parser {
+    public static List<Integer> parseNumbers(String numbers) {
+        return Arrays.stream(numbers.split(""))
+                .map(Integer::parseInt)
+                .collect(Collectors.toList());
```

Comment:
자바 17문법에 의해서 해당 부분은 아래와 같이 아무 간단하게 수정이 가능합니다!
```suggestion
                .toList();
```

참고하시면 도움이 될 것 같습니다!!

Reply:
업데이트된 문법에 대해서는 신경을 못 썼는데, 최신 버전을 적용하시는 모습에 감탄하고 갑니다. 
감사합니다.